usr/lib/common/loadsave.c

```diff
@@ -1205,10 +1269,9 @@ CK_RV save_masterkey_so_old(STDLL_TokData_t *tokdata)
     //
     // probably ought to ensure the permissions are correct
     //
-    sprintf(fname, "%s/MK_SO", tokdata->data_store);
-    fp = fopen(fname, "w");
+    fp = open_token_data_store_path(fname, sizeof(fname), tokdata, "MK_SO",
+                                    "w");
```

Comment:
the linux kernel now allows 100 characters per line if it is better for the readability

Reply:
That line is now 81 characters...